resources/views/controller.blade.php

```diff
@@ -233,6 +236,62 @@ public function destroy(Destroy{{ $modelBaseName }} $request, {{ $modelBaseName
         return redirect()->back();
     }
 
+
+    @if(!$withoutBulk)/**
+    * Remove the specified resources from storage.
+    *
+    * {{'@'}}param  Destroy{{ $modelBaseName }} $request
+    * @return Response|bool
+    * @throws \Exception
+    */
+    public function bulkDestroy(Destroy{{ $modelBaseName }} $request) : Response
+    {
+        $bulkChunked = array_chunk($request->data['ids'], 1000, true);
+
+        DB::transaction(function () use ($bulkChunked){
+            collect($bulkChunked)->each(function($bulkItem){
```

Comment:
$bulkItem is not the best name considering that it is not a single item.

Reply:
Agree with Samuel one this one, bulkChunk is definitely better name